src/main/blackbox/blackbox.c

```diff
@@ -1422,7 +1422,7 @@ static bool blackboxWriteSysinfo(void)
 #ifdef USE_GYRO_DATA_ANALYSE
         BLACKBOX_PRINT_HEADER_LINE("dyn_notch_max_hz", "%d",                gyroConfig()->dyn_notch_max_hz);
         BLACKBOX_PRINT_HEADER_LINE("dyn_notch_count", "%d",                 gyroConfig()->dyn_notch_count);
-        BLACKBOX_PRINT_HEADER_LINE("dyn_notch_bandwidth_hz", "%d",          gyroConfig()->dyn_notch_bandwidth_hz);
+        BLACKBOX_PRINT_HEADER_LINE("dyn_notch_q", "%d",                     gyroConfig()->dyn_notch_q);
```

Comment:
This will also need coordination with the blackbox log viewer - have these changes already been factored into the work that @ctzsnooze is doing there?

Reply:
I’m setting blackbox log headers to display Q values, rather than bandwidth, for rpm and dynamic notch filters - so that’s all ok, I think.